test/irb/yamatanooroti/test_rendering.rb

```diff
@@ -80,7 +80,7 @@ def b; true; end
       irb(main):008:0>
       irb(main):009:0> a
       irb(main):010:0>  .a
-      irb(main):011:0>  .b
```

Comment:
This indent was not a feature but a bug.

Paste this code and press return key
```ruby
a
  .b
  .c
if d
```
then you will get
```
irb(main):001:0> a
irb(main):002:0>   .b
irb(main):003:0>   .c
irb(main):004:1*   if d
irb(main):005:1*     
```